.template-lintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     extends: 'recommended',
 
     rules: {
-        'block-indentation': 4
+        'block-indentation': 4,
+        'bare-strings': false
```

Comment:
In theory, we want ember-osf to have full translation support, to best support consuming apps (like preprints) that also have this requirement.

Is there a reason why [bare-strings](https://github.com/rwjblue/ember-template-lint#bare-strings) is disabled here? Is this a permanent or temporary change?

(Temporary is fine. The deprecation warnings are annoying)